content/the-ember-times-issue-180.md

```diff
@@ -62,15 +64,15 @@ Check out Fleetbase on [ProductHunt](https://www.producthunt.com/posts/fleetbase
 
 ---
 
-## [4. Section title in sentence case 🐹](section-url)
-
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## New shiny add-ons ✨
+  
+The Ember add-on ecosystem is always expanding and we love to keep you up to date with all of them. So here is a summarized list of the latest add-ons made by the community for the community.
```

Comment:
I think addon is usually spelled without a hyphen in the context of Ember.js. (The difference isn't too important.)

```suggestion
## New shiny addons ✨
  
The Ember addon ecosystem is always expanding and we love to keep you up to date with all of them. So here is a summarized list of the latest add-ons made by the community for the community.
```